zproject/dev_settings.py

```diff
@@ -12,13 +12,20 @@
 DEPLOY_ROOT = os.path.realpath(os.path.dirname(os.path.dirname(__file__)))
 LOCAL_UPLOADS_DIR = os.path.join(DEPLOY_ROOT, 'var/uploads')
 
+def is_dev_droplet() -> bool:
```

Comment:
This broke run-dev because `zthumbor` imports this from Python 2.

```
Traceback (most recent call last):
  File "/srv/zulip-thumbor-venv/bin/thumbor", line 8, in <module>
    sys.exit(main())
  File "/srv/zulip-venv-cache/87ec5a9b1be44afb6c140ba27c6cd627dac950f3/zulip-thumbor-venv/local/lib/python2.7/site-packages/thumbor/server.py", line 145, in main
    config = get_config(server_parameters.config_path, server_parameters.use_environment)
  File "/srv/zulip-venv-cache/87ec5a9b1be44afb6c140ba27c6cd627dac950f3/zulip-thumbor-venv/local/lib/python2.7/site-packages/thumbor/server.py", line 53, in get_config
    return Config.load(config_path, conf_name='thumbor.conf', lookup_paths=lookup_paths)
  File "/srv/zulip-venv-cache/87ec5a9b1be44afb6c140ba27c6cd627dac950f3/zulip-thumbor-venv/local/lib/python2.7/site-packages/derpconf/config.py", line 79, in load
    return cls.__load_from_path(conf, path)
  File "/srv/zulip-venv-cache/87ec5a9b1be44afb6c140ba27c6cd627dac950f3/zulip-thumbor-venv/local/lib/python2.7/site-packages/derpconf/config.py", line 99, in __load_from_path
    six.exec_(code, module.__dict__)
  File "/srv/zulip-venv-cache/87ec5a9b1be44afb6c140ba27c6cd627dac950f3/zulip-thumbor-venv/local/lib/python2.7/site-packages/six.py", line 719, in exec_
    exec("""exec _code_ in _globs_, _locs_""")
  File "<string>", line 1, in <module>
  File "<string>", line 27, in <module>
  File "/srv/zulip/zproject/dev_settings.py", line 15
    def is_dev_droplet() -> bool:
                         ^
SyntaxError: invalid syntax
```